.github/workflows/electron.yaml

```
@@ -17,10 +17,12 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v1
```

actionlint (1.7.11)

[error] the runner of "actions/checkout@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)